src/test/initialize.ts

```diff
@@ -25,6 +26,11 @@ process.env.VSC_JUPYTER_CI_TEST = '1';
 // Ability to use custom python environments for testing
 export async function initializePython() {
     await setPythonPathInWorkspaceRoot(PYTHON_PATH);
+    // Make sure the python extension can load if this test allows it
+    if (!process.env.VSC_JUPYTER_CI_TEST_DO_NOT_INSTALL_PYTHON_EXT) {
+        const extension = vscode.extensions.getExtension(PythonExtension)!;
+        await extension.activate();
```

Comment:
We shouldn't do this, we should let python extension get activated normally in code.
I.e. if its installed and we need it, we activate it (in jupytre ext) & if not installed, then we don't activate it.
This new code will force the python extension to activate before jupyter ext launches and we don't want that, as its not a realistic situation (could hide other bugs).

Happy to leave this until we figure out whats going on with the tests though.

Reply:
Yeah I did this because I wanted to make sure that wasn't causing the problem. Happy to move it back after I get it all passing